src/openforms/forms/tests/test_import_export.py

```diff
@@ -768,3 +768,66 @@ def test_export_with_filetype_information(self, m_get_solo):
         export_data = form_to_json(form.pk)
 
         self.assertIsInstance(export_data, dict)
+
+    @tag("gh-3182")
+    def test_import_form_without_cosign_tag(self):
+        resources = {
+            "forms": [
+                {
+                    "active": True,
+                    "authentication_backends": [],
+                    "is_deleted": False,
+                    "login_required": False,
+                    "maintenance_mode": False,
+                    "name": "Test Form",
+                    "internal_name": "Test Form",
+                    "product": None,
+                    "show_progress_indicator": True,
+                    "slug": "test-form",
+                    "url": "http://testserver/api/v2/forms/324cadce-a627-4e3f-b117-37ca232f16b2",
+                    "uuid": "324cadce-a627-4e3f-b117-37ca232f16b2",
+                    "confirmation_email_template": {
+                        "content": "Some content {% appointment_information %}\n{% payment_information %}",
+                        "subject": "A subject",
+                    },
```

Comment:
the test should include a realistic `translations` structure here and make assertions on that